kie-identity-session-provider/src/main/java/org/kie/provider/impl/SafeSessionInfoProvider.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.provider.impl;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.enterprise.context.SessionScoped;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import org.jboss.errai.security.shared.api.Role;
+import org.jboss.errai.security.shared.api.RoleImpl;
+import org.jboss.errai.security.shared.api.identity.User;
+import org.jboss.errai.security.shared.api.identity.UserImpl;
+import org.kie.internal.identity.IdentityProvider;
+import org.kie.provider.SessionInfoProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.uberfire.rpc.SessionInfo;
+import org.uberfire.rpc.impl.SessionInfoImpl;
+
+@SessionScoped
+public class SafeSessionInfoProvider implements SessionInfoProvider, Serializable  {
+
+    /** Generated serial version UID */
+    private static final long serialVersionUID = 8510219062936244657L;
+
+    private static final Logger logger = LoggerFactory.getLogger( SafeIdentityProvider.class );
```

Comment:
A copy&paste error from past I think 

```suggestion
    private static final Logger logger = LoggerFactory.getLogger( SafeSessionInfoProvider.class );
```